test/e2e/utils.go

```diff
@@ -38,7 +38,7 @@ import (
 
 var (
 	retryInterval        = time.Second * 5
-	timeout              = time.Minute * 5
+	timeout              = time.Minute * 2
```

Comment:
I wonder if it would be helpful to use an env var, and fallback to a default of 2 minutes...